src/components/EmailSubscription/index.view.tsx

```diff
@@ -70,13 +81,28 @@ const EmailSubscriptionForm: React.FC = () => {
           </div>
         </form>
       )}
-      {showFeedback && (
-        <div className="EmailSubscription__feedbackContainer">
-          <span className="EmailSubscription__feedbackMessage">
-            {feedbackMessage}
-          </span>
+      <div className="EmailSubscription__feedbackContainer">
+        <div className="EmailSubscription__icon">
+          {requestState === states.LOADING && (
+            <ClipLoader size={25} color={"#4f728e"} loading={true} />
+          )}
+          {requestState === states.SUBMITTED && (
+            <img
+              src="https://icongr.am/material/check-circle-outline.svg?size=25&color=4f728e"
```

Comment:
I would refrain from linking to an actual URL outside of the scope of the project. Let's try and rendering from within our repository. Is there a way to download these assets or possibly use an Icon API?